internal/backend/rclone/config.go

```diff
@@ -12,13 +13,15 @@ type Config struct {
 	Program     string `option:"program" help:"path to rclone (default: rclone)"`
 	Args        string `option:"args"    help:"arguments for running rclone (default: serve restic --stdio --b2-hard-delete)"`
 	Remote      string
-	Connections uint `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	Connections uint          `option:"connections" help:"set a limit for the number of concurrent connections (default: 5)"`
+	Timeout     time.Duration `option:"timeout"     help:"set a timeout limit to wait for rclone to establish a connection (default: 1m)"`
 }
 
 var defaultConfig = Config{
 	Program:     "rclone",
 	Args:        "serve restic --stdio --b2-hard-delete",
 	Connections: 5,
+	Timeout:     1,
```

Comment:
that's not 1 minute, but rather 1 nanosecond (after fixing the square seconds above).

Reply:
You are correct, fixed